frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -147,19 +147,27 @@ const flavorUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) =>
 
 const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
```

Comment:
I think we could make the logic a bit more reconcilable here.

Let's do two updaters

osUpdater
 - called first
 - resolves the `MOUNT_WINDOWS_GUEST_TOOLS` field according to the os (win, etc)
   - should set value (false if linux for example) and isHidden

winToolsUpdater
 - called after osUpdater
 - should create storage if `MOUNT_WINDOWS_GUEST_TOOLS` value is true but storage does not exist
 - should remove storage if `MOUNT_WINDOWS_GUEST_TOOLS` is false and storage exists

Reply:
done :+1: